web/modules/custom/bnf/bnf_client/bnf_client.module

```diff
@@ -79,12 +82,40 @@ function bnf_client_form_node_form_submit(array $form, FormStateInterface $form_
   catch (\Throwable $e) {
     \Drupal::messenger()->addError(t('Could not export node to BNF.', [], ['context' => 'BNF']));
 
-    if ($e instanceof AlreadyExistsException) {
-      \Drupal::messenger()->addError(t('Node already has been exported to BNF.', [], ['context' => 'BNF']));
-    }
-    else {
-      \Drupal::logger('bnf_client')->error('Could not export node to BNF. @message', ['@message' => $e->getMessage()]);
-    }
+    \Drupal::logger('bnf_client')->error('Could not export node to BNF. @message', ['@message' => $e->getMessage()]);
 
   }
 }
+
+/*
+ * The following hook functions is for Drupal 10, they can be removed, together
+ * with the hook class service in `bnf_client.services.yml` when upgrading to
+ * Drupal 11.
+ */
+
+/**
+ * Implements hook_ENTITY_TYPE_insert().
+ */
+#[LegacyHook]
```

Comment:
Does this automatically notify the developer that updates to Drupal 11?

Reply:
No, but it makes D11 ignore the hook function and use the hook class. Without it I guess the hook would be invoked twice in D11.